### ISSUE TITLE
Accessibility of q-img component

### ISSUE THREAD

**Issue by Morialkar** [quasarframework/quasar#6570]
**Describe the bug**
When using the native-context-menu prop, the component prints an <img> tag, which is what we need, but the alt prop doesn't cascade to the <img> tag, which makes it unusable for accessible images. Also, when doing so, the Aria-label prop of the parent element should be removed to not duplicate reading on a screen reader.

One of the main reason is that as the img tag has no alt prop, screen readers will read the file name, not skip it.

**Codepen/jsFiddle/Codesandbox (required)**
https://codepen.io/Morialkar/pen/KKpyaeO

**To Reproduce**
1. Create a new Quasar project
2. add a q-img component
3. add an alt prop
4. add the "native-context-menu" prop

**Expected behavior**
The alt prop should cascade to the <img> tag if you add the native-context-menu prop, allowing the use of the component and accessibility with screen readers and remove the the aria-label to remove duplicate data. An empty string should also be cascaded to allow skipping by the screen reader.

**Platform (please complete the following information):**
OS: macOS
Node: 12.13.0
NPM: 6.13.7
Yarn: 1.22.0
Browsers: Chrome

**Comment by pdanpdan** [comment]
I think the more normal solution would be to set `aria-hidden` on the img inside - would this change anything in your request?

**Comment by rstoenescu** [comment]
Fix will be available in “quasar” v1.11.2
